site_src/pages/home.ts

```
@@ -302,6 +304,35 @@ export function HomePage(opts: {
+            <div class="value">${numSpan(profile.aiUsageWeekly)}</div>
+            <div class="label" style="margin-top:0.25rem">of 1,000,000 tokens</div>
+          </div>
+          <div class="me-card">
+            <div class="label">Status</div>
+            <div class="value" style="font-size: 1.5rem; font-weight: 600;">
+              ${profile.aiUsageDaily >= 250000 || profile.aiUsageWeekly >= 1000000 
+                ? html`<span style="color: #ef4444;">Rate Limited</span>` 
+                : html`<span style="color: #10b981;">Active</span>`}
+            </div>
+            <div class="label" style="margin-top:0.25rem">
+              ${profile.aiUsageDaily >= 250000 ? 'Daily limit exceeded' : (profile.aiUsageWeekly >= 1000000 ? 'Weekly limit exceeded' : 'Token pool is cool')}
```

ESLint

[error] Do not nest ternary expressions.

(no-nested-ternary)